betel/classifier_data_set_builder.py

```diff
@@ -0,0 +1,65 @@
+import shutil
+import pathlib
+import pandas as pd
+from betel import utils
+from betel import info_files_helpers
+from betel import data_set_builder
+
+
+class ClassifierDataSetBuilder(data_set_builder.DataSetBuilder):
```

Comment:
could you remind me why do we need a base class here?

Reply:
I was thinking of another DataSetBuilder subclass for the image similarity model.